xcm/pallet-xcm/src/mock.rs

```diff
@@ -193,6 +195,38 @@ impl SendXcm for TestSendXcmErrX8 {
 	}
 }
 
+parameter_types! {
+	pub Para3000: u32 = 3000;
+	pub Para3000Location: MultiLocation = Parachain(Para3000::get()).into();
+	pub Para3000PaymentAmount: u128 = 1;
+	pub Para3000PaymentMultiAssets: MultiAssets = MultiAssets::from(MultiAsset::from((Here, Para3000PaymentAmount::get())));
+}
+/// Sender only sends to `Parachain(3000)` destination requiring payment.
+pub struct TestPaidForPara3000SendXcm;
```

Comment:
It feels like these items are specific to a particular test. Why not put them in `tests` rather than in mock? The test sender should be generic enough.